src/domain/common/haittaIndexes/HaittaSubSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Flex, Grid } from '@chakra-ui/react';
+import HaittaIndex from './HaittaIndex';
+import React from 'react';
+
+export function HaittaSubSection({
+  heading,
+  index,
+  oddEven = 'even',
+  testId,
+}: Readonly<{
+  heading: string;
+  index?: number;
+  oddEven?: 'odd' | 'even';
+  testId?: string;
+}>) {
+  return (
+    <Grid
+      templateColumns="1fr 24px"
+      gap="var(--spacing-xs)"
+      paddingX="var(--spacing-s)"
+      paddingY="var(--spacing-xs)"
+      background={oddEven === 'odd' ? 'var(--color-black-5)' : 'var(--color-black-10)'}
```

Comment:
Might be better to do the background color with css pseudo-classes. In that way there would be no need for `oddEven` prop. For example:
```
<Grid
  templateColumns="1fr 24px"
  gap="var(--spacing-xs)"
  paddingX="var(--spacing-s)"
  paddingY="var(--spacing-xs)"
  _odd={{ backgroundColor: 'var(--color-black-5)' }}
  _even={{ backgroundColor: 'var(--color-black-10)' }}
>
```

Reply:
I did not know about this, thanks! Fixed as you suggested.